docs/md_docs_2sources_2changelog.html

```diff
@@ -114,37 +114,40 @@
 </div><!--header-->
 <div class="contents">
 <div class="textblock"><h1>2.2.0</h1>
-<h2>July 15, 2023</h2>
+<h2>July 17, 2023</h2>
 <ul>
-<li><em>Vastly improved</em> support for syslog, if available, and macOS' system logger (os_log) as their own independent logging destination</li>
-<li>Support for 256-color and RGB color modes.</li>
 <li>Many new platforms and toolsets are now supported. See <a href="https://libsir.rml.dev">the main page</a> for details.</li>
-<li>Better and more thorough documentation.</li>
-<li><u>Massive performance gains</u> for both stdio and log files via optimization/refactoring (estimated 30%+)</li>
-<li><em>Lots</em> of work to get things going smoothly on Windows<ul>
-<li>A Visual Studio 2022 solution in <code>msvs</code></li>
+<li>Plugin loader and sample plugin module implemented! This gives libsir virtually unlimited potential, and in the near future, some standard plugins will be available as part of the library. See documentation for <code>sir_loadplugin</code> and <code>sir_unloadplugin</code>. The sample plugin is located at <code>plugins/sample/sample_plugin.c</code></li>
```

Comment:
OK once plugins is in, but I'd be more comfortable with more testing of that branch, and will have to run it by all the platforms.